src/style.css

```diff
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
It's better to avoid using direct tag selectors for styling. In this case, you are applying styles to the `body` tag. Although it's okay for some global styles, it's better to create a specific class for this. It improves the maintainability of the code and avoids unexpected issues in the future.